sklearn/ensemble/tests/test_bagging.py

```diff
@@ -878,3 +878,21 @@ def test_bagging_small_max_features():
     bagging = BaggingClassifier(LogisticRegression(),
                                 max_features=0.3, random_state=1)
     bagging.fit(X, y)
+
+
+def test_bagging_get_estimators_indices():
+    # Check that Bagging estimator can generate sample indices properly
+
+    X = np.zeros((13, 1))
```

Comment:
Just to not give an null dataset

```suggestion
    rng = np.random.RandomState(0)
    X = rng.randn((13, 4))
```

Reply:
Agree. I changed  the 2nd line into `X = rng.randn(13, 4)`.